libraries/chef_server_ingredients_provider.rb

```diff
@@ -74,6 +81,13 @@ def ctl_command
         new_resource.ctl_command || chef_server_ctl_command(new_resource.package_name)
       end
 
+      def reconfigure
+        ctl_cmd = ctl_command
+        execute "#{new_resource.package_name}-reconfigure" do
+          command "#{ctl_cmd} reconfigure"
```

Comment:
I assumed that you had it right so I just replicated, but I'm not sure if you can use ctl_cmd directly there or not.